recipes/py_gd/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.1.6" %}
+{% set libgd_version = "2.2.2" %}
+
+package:
+    name: py_gd
+    version: {{ version }}
+
+source:
+    git_url: https://github.com/NOAA-ORR-ERD/py_gd.git
+    git_tag: v{{ version }}
```

Comment:
Correct. Unless they provide it for us somehow. It is a bit undesirable we have to compute it ourselves, but that is the lay of the land.